IVRv2/app/main.py

```
@@ -857,6 +863,142 @@ async def dtmf(input: Request):
+                    "maxDigits": 1,
+                    "submitOnHash": False,
+                    "timeOut": 10
+                }
+            }])
+
+    # Handle timeout during WebSocket playback - keep listening without interrupting
+    if digits == "" and dtmf_input.dtmf.timed_out:
+        # Check if current state has WebSocket connection (VonageConnectAction)
+        current_state = fsm_in_progress.states.get(ivr_state.current_state_id)
+        if current_state and any(isinstance(action, VonageConnectAction) for action in current_state.actions):
+            logging.info(f"Input timeout during WebSocket playback - returning InputAction to continue listening")
```

Ruff (0.15.10)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)